### PR TITLE
Move pascal_voc --> voc

### PR DESCRIPTION
**Pull request by yuyu2172**
Change the directory name because `pascal_` is redundant.

### CODE REVIEW

**Comment by Hakuyume** [comment]
Currently, ChainerCV downloads the dataset under `.chainer/dataset/pfnet/chainercv/pascal_voc/`.
For consistency, is it better to change the destination?

**Comment by yuyu2172** [comment]
Right. Thanks for catching that.

File: docs/source/reference/datasets.rst [hunk 1]
```diff
@@ -32,8 +32,8 @@ OnlineProductsDataset
 .. autofunction:: OnlineProductsDataset
 
 
-PASCAL VOC
-----------
+VOC
```
Comment (Hakuyume) [inline docs/source/reference/datasets.rst]: I think `PASCAL VOC` is better for documents. How do you think of it?

Comment (yuyu2172) [inline docs/source/reference/datasets.rst]: OK.

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**